javascript/app.js

```diff
@@ -137,8 +144,23 @@ FinancialFreedom.controller('TimeToRetirementController', ['$scope', 'Retirement
     
 }]);
 
-FinancialFreedom.filter('percentage', ['$filter', function ($filter) {
-  return function (input, decimals) {
-    return $filter('number')(input * 100, decimals) + '%';
-  };
-}]);
+FinancialFreedom.directive("percent", function($filter){
+    var p = function(viewValue){ // format model value
+        var m = viewValue.match(/^(\d+)\/(\d+)/);

```

Comment:
I found this percentage directive online. Is that just javascript regex syntax (/regexhere/)?